Assets/Plugins/CommentToTooltip/ReadMe.txt

```diff
@@ -21,4 +23,11 @@ This asset is subject to Unity's standard Unity Asset Store End User License Agr
 
 Contact & Support
 =================
-If you have any questions, feedback or requests, please write to admin@curvyeditor.com 
\ No newline at end of file
+If you have any questions, feedback or requests, please write to admin@toolbuddy.net 

```

Comment:
kinda started this branch based on the version of the source code that was provided with the version of the tool which was published on the unity asset store, and yeah the @toolbuddy.net email address was on the readme of the unity asset store version, so yeah when I kinda pushed the files from that into the fork, that caused this line to get changed.